recipes/libyuv/all/patches/001-cmake.patch

```diff
@@ -0,0 +1,55 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -6,6 +6,9 @@
+ CMAKE_MINIMUM_REQUIRED( VERSION 2.8 )
+ OPTION( TEST "Built unit tests" OFF )
+ 
++include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
++conan_basic_setup()
++
+ SET ( ly_base_dir	${PROJECT_SOURCE_DIR} )
+ SET ( ly_src_dir	${ly_base_dir}/source )
+ SET ( ly_inc_dir	${ly_base_dir}/include )
+@@ -22,13 +25,19 @@
+ 
+ INCLUDE_DIRECTORIES( BEFORE ${ly_inc_dir} )
+ 
++if(MSVC)
++    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
++endif()
```

Comment:
```suggestion
+if(MSVC)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+endif()
```

Export it to a CMakeLists.txt, it's a common pattern here in CCI: https://github.com/conan-io/conan-center-index/blob/ccd4ba40a2f39d03108c3bb25bae80c1410e8114/recipes/kcp/all/CMakeLists.txt

Reply:
Fixed